main.md

```diff
@@ -164,6 +164,47 @@ Figure: SD-JWT Issuance and Presentation Flow
 
 # Concepts
 
+From the point of view of the collection limitation, which is one of the main privacy principles, it is desirable to be able to provide only the claims necessary for the verifier to process. In the case of a standard JWT [RFC7519], however, this can only be achieved when the JWT is minted afresh each time and it is not possible to re-use the JWT that is pre-minted for other occasions. For example, a holder may want to re-use the document that contains name, address (municipality, street address, room number), date of birth, and National Identification Number to prove that he is a resident of the municipality so that he can receive the service to the resident. In the case of the normal JWT, this is not possible as the signature is over the entire claim set directly. To achieve it, a JWT that only contains the claim about the municipality needs to be minted. 
```

Comment:
How is this text different from what is already in "the introduction"? I would rather have this text as a new "use-cases" section.

Also isn't "data minimization" a more common term than "collection limitation"?